src/command_line_parser.cc

```diff
@@ -1777,6 +1830,11 @@ CLParser::ParseCommandLine(int argc, char** argv)
     }
   }
 
+  if (params_->inference_load_mode == InferenceLoadMode::None ||
+      using_deprecated_concurrency) {
+    params_->inference_load_mode = InferenceLoadMode::Concurrency;
+  }
+
```

Comment:
This enforces the usual default mode of concurrency.

params_->inference_load_mode needed to hold `None` mode to detect repeated usage, so we set actual default here.